docs/SEISMIC_SENSOR_ORIENTATION_POLARITIES.md

```diff
@@ -0,0 +1,61 @@
+# Seismic Sensor Orientation and Polarity Conventions and Methods
+
+The conventions and methods used by GeoNet to measure and represent seismic sensor orientations and polarities are outlined in this document.
+
+Data users encounter orientations and polarities through the component orientations in StationXML. This document describes the underlying conventions and processes which define these values.
+
+## Orientation Conventions and Measurement Methods
+
+Sensor orientations are contained in `delta.git/install/sensors.csv` under the `Azimuth` column. All sensor installations have an orientation entry, although it is not always defined.
+
+Seismic sensor orientations indicate the bearing of the sensor's north component and assume a left-hand convention for the orientation of the vertical and east components with respect to the north component (i.e., E component is at 90 degrees clockwise of N in the N-E plane; Z component is 90 degrees anticlockwise in N-Z plane).
+
+Sensors that are not *intended* to be north-aligned will have `1` and `2` as the last character of their horizontal component channel code, and sensors that are intended to be north-aligned will be `N` and `E` as the last character of their horizontal component channel code - as per `docs/MINISEED_METADATA_CONVENTIONS.md`. It is important to mark this distinction: that all sensors have an orientation, but not all sensors are intended to be north-oriented.
+
+### Unknown
+
+When an orientation is unknown, the `Azimuth` value will be *empty* and the `Method` will be *empty*. This ensures that azimuth information is not provided to data users who may otherwise take values indicative of an unknown orientation as the orientation itself.
+
+### Onsite
+
+When an orientation has been measured at the station, the `Azimuth` value will be this measurement and the `Method` will be `onsite`.
+
+The most common onsite orientation methods are using a compass and an azimuith pointing system. A compass measurement involves taking a bearing of the sensor's north component using a compass. An azimuth pointing system (APS) projects a laser onto the sensor's north component marking and uses differential GPS positioning of two antenna at either end of its laser to measure the sensor orientation. Both of these approaches are considered reliable and which method has been used to measure sensor orientation is not noted.
+
+### Offsite
+
+When an orientation is calculated using the data from a station or - as in the case of strong motion sensors in building arrays - using geographic references, the `Azimuth` value is the calculated value and the `Method` is `offsite`.
```

Comment:
Is the `offsite` for `Method` on of the available units for this metadata? If yes, we can include this in the README (https://github.com/GeoNet/delta/blob/main/install/README.md).

Reply:
All orientations methods produce values in degrees, which go into Azimuth columns so... yes?

Reply:
Ah yeah, apologies. I was looking at the _Method_ field in _polarities.csv_ . I'm not so familiar with this new metadata yet, so I'm exploring the units. From the README, Method in _polarities.csv_ has these options: "study", "compass", or "unknown"